c++/src/kj/encoding.h

```diff
@@ -197,6 +197,7 @@ EncodingResult<Array<byte>> decodeBinaryUriComponent(
 // Decode URI components using % escapes. This is a lower-level interface used to implement both
 // `decodeUriComponent()` and `decodeWwwForm()`
 
+String encodeCEscapeImpl(ArrayPtr<const byte> bytes, bool isBinary);
```

Comment:
This should be under `namespace _ {  // private` below.

Reply:
Adjusted & updated PR, thanks! I've also extended encodeCEscape(chars) with UTF-8 text and added small test for the encodeCEscape(bytes).